python/api_model/types/models.py

```
@@ -246,8 +246,11 @@ class InvalidRemove(RootModel[str]):
 
 
 class SharedUserDetails(BaseModel):
-    userId: str = Field(..., pattern='^[A-Za-z0-9-]+$')
-    email: str = Field(..., pattern='^[\\w-\\.]+@[\\w-\\.]+\\.+[\\w-]{1,63}$')
+    sharedWithUserId: str = Field(..., pattern='^[A-Za-z0-9-]+$')
+    sharedByUserId: str = Field(..., pattern='^[A-Za-z0-9-]+$')
+    sharedWithUserEmail: str = Field(
+        ..., pattern='^[\\w-\\.]+@[\\w-\\.]+\\.+[\\w-]{1,63}$'
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)